runtime/compilers/rillv1/util.go

```diff
@@ -0,0 +1,29 @@
+package rillv1
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ProjectConfig struct {
+	// Project variables
+	Variables   map[string]string `yaml:"env,omitempty"`
+	Title       string            `yaml:"title,omitempty"`
+	Description string            `yaml:"description,omitempty"`
+}
+
+func HasRillProject(dir string) bool {
+	_, err := os.Open(filepath.Join(dir, "rill.yaml"))
+	return err == nil
+}
```

Comment:
The goal is that this package doesn't do direct file system reads and instead using the `drivers.RepoStore` interface. 

So the callers should use something like `cmdutil.RepoForProjectPath` and then this function should be changed to accept a repo (i.e. have a type signature similar to `rillv1.ParseRillYAML(...)`

Reply:
like this? 
```
func HasRillProject(dir string) bool {
	repo, _, err := RepoForProjectPath(dir)
	_, err = repo.Get(context.Background(), "rill.yaml")
	return err == nil
}
```